draft-ietf-ppm-dap.md

```diff
@@ -1960,7 +1960,7 @@ struct {
 struct {
   opaque agg_param<0..2^32-1>;
   PartialBatchSelector part_batch_selector;
-  PrepareInit prepare_inits<44..2^32-1>;
+  PrepareInit prepare_inits[prepare_inits_length];
```

Comment:
It might be nice to use the same variable name for all messages. What about "suffix_length" to denote that this is supposed to be the length of the rest of the body? It also fits nicer with "message_length".

Reply:
If we used `suffix_length`, then we'd have to explain what we mean by "suffix" which unhelpfully complicates the explanatory text that follows. I think it's perfectly clear that `prepare_inits_length` is the length of `prepare_inits`, and then we explain in prose how to compute `prepare_inits_length`.